test/Spec/BotPlutusInterface/Contract.hs

```diff
@@ -108,7 +108,7 @@ sendAda = do
               Constraints.mustPayToPubKey paymentPkh2 (Ada.lovelaceValueOf 1000)
         submitTx constraints
 
-  assertContractWithTxId contract initState $ \state outTxId ->
+  assertContractWithTxId contract initState $ \state _ ->
```

Comment:
[nits] We don't need to use `assertContractWithTxId` anymore, this could be replaced with a simple `runContractPure`